src/daemon/control.rs

```diff
@@ -726,10 +740,212 @@ pub fn watchtowers_status(revaultd: &RevaultD) -> Vec<ServerStatus> {
             });
         }
     }
-
     watchtowers
 }
 
+// FIXME: probably this function should be in revault_tx, not here...
```

Comment:
Yes.

Reply:
Is this a "yes, let's do it now" or a "yes, we'll move it" eventually?